std-bits/base/src/main/java/org/enso/base/enso_cloud/AuthenticationProvider.java

```diff
@@ -16,7 +16,10 @@ public static String getToken() {
 
   public static String getAPIRootURI() {
     var envUri = Environment_Utils.get_environment_variable("ENSO_CLOUD_API_URI");
-    return envUri == null ? "https://7aqkn3tnbc.execute-api.eu-west-1.amazonaws.com/" : envUri;
+    var effectiveUri =
+        envUri == null ? "https://7aqkn3tnbc.execute-api.eu-west-1.amazonaws.com/" : envUri;
```

Comment:
Could be a constant

Reply:
It couldn't, because we sometimes override the environment variable in tests. We would be unable to do so if it was baked-in at initialization time.